### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mahbubul14**
### This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux.

- [Api for retrive the data](https://covid19tracking.narrativa.com/index_en.html)
- [Loom video for presentation](https://loom.com/share/b81cbe21ee0e4be2b51a925770980896)
- Created the tests for the application and completed all other requirements for the project

### Chossing API

Some example APIs are:
* [ Financial modeling prep API](https://site.financialmodelingprep.com/developer/docs/): stock data.
* [Narrativa API:](https://covid19tracking.narrativa.com/index_en.html) COVID-19 data.
* [Air pollution API:](https://openweathermap.org/api/air-pollution) air quality **data.**

You can find many APIs in [this GitHub repo](https://github.com/public-apis/public-apis) or in [ProgrammableWeb](https://www.programmableweb.com/category/all/apis). Some of the APIs require a token for authentication and some others are just open.

### Design

* You should follow these [design guidelines](behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
    * Colors (select one main color for your website).
    * Typography: font face, size and weight.
    * Layout: composition and space between elements.

### Interactions:
  * Home page
 
     * When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
     * Along with the category name, you will display some numeric values per category that come from the API.
     * When the user clicks (or taps) on a category item, the application navigates to the details page.\
     
  * Details page
     *  In the details page, the webapp retrieves data from the API to show detailed data in the category.
     * When the user clicks on the "Back" button (<), the user navigates to the home page.

### Testing requirements

 * The project is a single page application (SPA) built with React and Redux.
 * The data retrieved from the API should be stored in the Redux store.
 * You should filter the data that you retrieve from the API using a Filter stateless component.
 * Every page (the main page and the pages for each item) should have a unique route within the SPA.
 * The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+## Built With
+
+- React
+- JSX
+- Redux
+- [CORONAVIRUS DATA API - COVID-19 TRACKING PROJECT](https://covid19tracking.narrativa.com/index_en.html)
+
+## Live Demo
+
+- N/A
```
Comment (GzimAsani) [inline README.md]: - [x] You should add a live demo of your capstone, you can use Heroku, Netlify or GitHub Pages

File: src/__tests__/ComponentsTesting/App.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```
Comment (GzimAsani) [inline src/__tests__/ComponentsTesting/App.test.js:1]: - [x] Some of your tests are failing kindly fix them!
Screenshot from 2021-11-25 20-20-59

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @mahbubul14                            
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @mahbubul14 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**